configuration/README.md

```diff
@@ -8,7 +8,9 @@ Some basic guides to configuring your Raspberry Pi.
     - The Raspberry Pi configuration tool in Raspbian, allowing you to easily enable features such as the camera, and to change your specific settings such as keyboard layout
 - [config.txt](config-txt/README.md)
     - The Raspberry Pi configuration file
-- [Wireless networking](wireless/README.md)
+- [TCP/IP networking](ethernet/README.md)
```

Comment:
Should the directory filename be changed from `ethernet` to `tcpip` or `dhcpcd` ?

Reply:
> Should the directory filename be changed from `ethernet` to `tcpip` or `dhcpcd` ?

Indeed it should. I was avoiding doing that to prpeserve backwards compatibility, but since the file is new as of this PR that's not an issue. I've gone for 'tcpip' as a directory name.